.github/workflows/dotnetcore.yml

```diff
@@ -16,3 +16,6 @@ jobs:
 
     - name: Build with dotnet
       run: dotnet build --configuration Release
+
+    - name: Run test suite
+      run: dotnet test --configuration Release --logger "console;verbosity=normal"
```

Comment:
I'd _like_ to enable color output here in CI, but I'm not fully sure about how to do it. The https://github.com/Microsoft/vstest/blob/main/src/vstest.console/Internal/ConsoleLogger.cs URL is the link to the `ConsoleLogger` being used here, but I suspect this is more of a flag to `dotnet test` or something. (to ensure `ConsoleLogger` gets instantiated with an `IOutput` which supports color.)

Reply:
Let's take this in a separate PR, merging away. :rocket:

Reply:
Update: xunit v3 (which _might_ make this easier) is not really ready for public usage yet. I'll postpone this until we can go for the xunit 3.0 upgrade at some point.